debian/changelog

```diff
@@ -12,10 +12,14 @@ cloud-init (25.1-0ubuntu1~24.04.1) UNRELEASED; urgency=medium
     - d/p/no-nocloud-network.patch
     - d/p/deprecation-version-boundary.patch
     - d/p/no-single-process.patch
+  * add d/p/strip-invalid-mtu.patch
+    - Provides backwards compatibility for an otherwise invalid
+      MTU in a netplan config.
   * sort hunks within all patches (--sort on quilt refresh)
-  * Upstream snapshot based on upstream/main at ff1d9f04.
+  * Upstream snapshot based on upstream/main at 2b6c7e65.
+  * TODO: MOVE IN 25.1.x changes
```

Comment:
Sure, this works, it's a good gate in UNRELEASED changelog entry. I've already included all changes from 25.1.x as of July 8th's new_upstream_snapshot to fix daily build recipes. But it's still worth leaving that breadcrumb around in case something changes in security review/guidance (which I don't expect).

Reply:
Shoot. that's because it's not there. ok I sync'd ChangeLog not d/changelog. You are correct.